### PR TITLE
Fix phpunit issue

### PR DESCRIPTION
**Pull request by dereuromark**
Dont use the broken global one

Fixes at least the 5.6 fails.

### CODE REVIEW

**Comment by ravage84** [comment]
Wouldn't ``composer  test`` work, if we had a ``test``script defined under the ``script`` key in ``composer.json``?

**Comment by dereuromark** [comment]
It might, but thats not for travis IMO, thats for end user "usage" :)

File: composer.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     },
     "require-dev": {
         "cakephp/cakephp-codesniffer": "~2.1",
-        "phpunit/phpunit": "5.*"
+        "phpunit/phpunit": "<6.0"
```
Comment (ravage84) [inline composer.json]: Would the tests work with PHPUnit 3.7? 😼
I think not. So something like ``>=4.0 <6.0`` would be _more_ correct.

**Comment by antograssiot** [comment]
All tests passing now

File: composer.json (hunk 1 above)
Comment (antograssiot) [inline composer.json]: What's the point ? I don't understand as composer will always fetch the latest version possible and debug_kit is php 5.5...

**Pull request merged**